docs/add_content.rst

```diff
@@ -350,6 +352,8 @@ To create an exercise, follow these steps.
 
          Add Prerequisites for the current exercise.
 
+   .. warning:: Currently the **Prerequisites** category is used to merely indicate the **recommended content** that will allow learners to revisit key prior concepts, foundational skills, or immediately relevant background information. For learners on Kolibri, these items will appear alongside the concept for recommended viewing, **but will not be compulsory** in order to view the current resource.
```

Comment:
To note: a user can set any resource type as a prerequisite in addition to videos/exercises